libs/platform/src/lib/components/form/multi-input/multi-input.component.scss

```diff
@@ -1,101 +1,19 @@
-//TODO evaluate whether this should be in fd-styles
+//Cutomization based on the width of the input used
```

Comment:
Don't we need to import the css from styles?

Reply:
these styles are not part of fundamental style, these are specifically needed for the multi input component like how its used in core  libs/core/src/lib/multi-input/multi-input.component.scss